postreise/plot/plot_capacity_map.py

```diff
@@ -31,13 +31,13 @@ def map_plant_capacity(
     :param powersimdata.scenario.scenario.Scenario scenario: scenario instance.
     :param iterable resources: which types of resources to plot.
     :param tuple figsize: size of the bokeh figure (in pixels).
-    :param tuple(float, float) x_range: x range to zoom plot to (EPSG:3857).
-    :param tuple(float, float) y_range: y range to zoom plot to (EPSG:3857).
+    :param tuple x_range: x range to zoom plot to (EPSG:3857).
+    :param tuple y_range: y range to zoom plot to (EPSG:3857).
     :param str disaggregation: method used to disaggregate plants:
         if "new_vs_existing_plants": separates plants into added vs. existing.
         if None, no disaggregation.
-    :param dict plot_states_kwargs: keyword arguments to pass to
-        :func:`postreise.plot.plot_states.plot_states`.
+    :param dict state_borders_kwargs: keyword arguments to pass to
```

Comment:
Is there a way to validate this input? It might be helpful to return a more direct error if `state_borders_kwargs` contains an entry that isn't a valid keyword argument. You could use the `inspect` module to get a list of all the keyword arguments currently defined in the `add_state_borders` module:
```
import inspect
valid_keywords = inspect.getfullargspec(add_state_borders).args
```

Reply:
Nice. I will implement this in functions passing arguments to other.

Reply:
Done